### ISSUE TITLE
Not really a issue instructions are a bit unclear

### ISSUE THREAD

**Issue by BoxDevelopment** [CoolSpy3/CSModLoader#4]
Hello,

I saw the CSModLoader and i wanted to try to load some forge mods into lunar Issue is i have the mental age of a 5 year old,

Is it possible some instructions on how to use the .jar can be made

Thanks in advanced

**Comment by CoolSpy3** [comment]
Hello,

Sorry for not getting back sooner. I've updated the README to include some instructions on installation. However, I'm unsure if the loader is suitable for your use case. I touched on it in the previous version of the README, but the loader can't directly load forge mods due to the constraints of its design. Most simpler mods could probably be ported without too much work (This is what I did with [Sk1er's AutoGG Mod](https://sk1er.club/mods/autogg) which I ported to [CSAutoGG](https://github.com/CoolSpy3/CSAutoGG)), but anything more complex is beyond my expertise.
